docs/glossary.md

```diff
@@ -0,0 +1,62 @@
+**Acronyms**: CatSrc(s)
+
+### ClusterServiceVersion
+
+**Definition**: The ClusterServiceVersion is a required resource for an Operator which includes application metadata such as name, description, version, repository link, labels, icon, etc. It also includes a list of corresponding/required resources and their statuses, install strategy of the Operator as [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), supported install modes, etc.
+**Acronyms**: CSV(s)
+
+### InstallPlans
+
+**Definition**: The InstallPlan lists out all resources to be created in order to automatically install or upgrade a [CSV](#ClusterServiceVersion). 
+**Acronyms**: IP(s)
+
+### OperatorGroups
+
+**Definition**: The operator group selects a set of target namespaces in which to generate required RBAC access for its member Operators.
+**Acronyms**: OG(s)
+
+### OperatorSources
+
+**Definition**: The operator source defines the external data store used to store Operator bundles. It contains information about the catalog of operators including its name, type, namespace, and display name. 
+**Acronyms**: OpSrc(s)
+
+### Subscriptions
+
+**Definition**: The subscription defines the channel and the source of Operator updates and is used to keep the CSV updated by tracking a channel in a package. 
+**Acronyms**: Subs(s)
+
+## OLM Concepts
+
+### Bundle
+
+**Definition**: A collection of Operator manifests and metadata which together form a unique version of an Operator that can be installed onto the cluster. 
+**Acronyms**: Package
+
+### Catalog Image
+
+**Definition**: A catalog image is an image that contains Operator bundles that are stored in a database. These are then exposed over gRPC to the OLM Operator to allow users to query and install new Operators onto the cluster. 
+**Acronyms**: OPM Index
+
+### Index
+
+**Definition**: The Index refers to a database that contains information about Operator bundles including CSVs, CRDs, etc of all versions. This index can host a history of used operators on a cluster and be maintained by adding or removing operators.
+**Acronyms**: Registry DB, Catalog DB, OPM registry
+
+### Upgrade Graph
+
+**Definition**: An upgrade graph links versions of [CSV](#ClusterServiceVersions) together, similar to the upgrade graph of any other packaged software. Operators can be installed sequentially, or certain versions can be skipped. The update graph is expected to grow only at the head with newer versions being added. 
+**Acronyms**:
+
+### Dependency
+
+**Definition**: An Operator may have a dependency on another Operator being present in the cluster. For example, the Vault Operator has a dependency on the Etcd Operator for its data persistence layer. OLM resolves these dependencies by ensuring all specified versions of Operators and CRDs are installed on the cluster during the installation phase. 
```

Comment:
This will be misleading to newcomers - we should mention that it's based on api dependencies and not related to packages/bundles.